R/partial-eval.R

```diff
@@ -70,7 +70,7 @@ partial_eval <- function(call, vars = character(), env = caller_env()) {
   }
 }
 
-partial_eval_dots <- function(dots, vars) {
+partial_eval_dots <- function(dots, vars, named = FALSE) {
```

Comment:
Currently, `partial_eval_dots()` does not name `across()` calls. The names are needed for `mutate()` and `transmute()` but e.g. for `filter()` the dots must not be named.